docs/developing/ui-component-testing.md

```diff
@@ -0,0 +1,34 @@
+# UI Component Testing
+
+This document gives you a process to follow to check that any UI components
+you have created will be in tip-top shape for the site.
+
+## Browsers to check
+
+It doesn't have to look exactly the same in every browser, but it should look similar and work properly in current versions of these browsers:
+
+### Desktop
+
+* Firefox
+* Opera
+* Chrome
+* Edge
+* IE 11
+* Safari (latest and latest but one)
+
+### Mobile
+
+Latest versions of:
+
+* iPhone: Chrome and Safari
+* iPad: Chrome and Safari
+* Samsung phone: Chrome and Firefox
+* Google phone: Chrome and Firefox
```

Comment:
There are many browsers (and older versions of browsers) that have higher usage than Firefox mobile's 0.2% user share.

Reply:
Good point. Replace Firefox with UC Browser.